### PR TITLE
python 3.11 support

### PR DESCRIPTION
**Pull request by Nathanjp91**
# Problem
darwin-py doesn't support python 3.11

# Solution
Change dependencies to allow 3.11 support

# Changelog
Python 3.11 support added. Python 3.7 support dropped

### CODE REVIEW

**Review by owencjones — DISMISSED** [review]
LGTM, some offline discussion to be had in house

File: pyproject.toml [hunk 1]
```diff
@@ -72,20 +72,20 @@ jsonschema = ">=4.0.0"
 deprecation = "^2.1.0"
```
Comment (owencjones) [inline pyproject.toml:72]: Due to [Github limitation](https://github.com/orgs/community/discussions/4452) I can't comment on the exact line I need to, but on `L69` can be entirely removed now, as it's only needed by Python 3.7

Arguably, we could keep it for people who forcibly install on 3.7, but I think I err on the side of not doing personally, especially given that you've changed other python version dependant installs.

**Review by owencjones — CHANGES_REQUESTED** [review]
I think remove `pyproject.toml:L69`, or at least consider it and respond. 🙂

**Review by owencjones — APPROVED** [review]

**Comment by Nathanjp91** [comment]
Working via this PR instead
https://github.com/v7labs/darwin-py/pull/692

**Pull request closed without merging**